components/dashboards-web-component/src/designer/components/WidgetOptionsConfiguration.jsx

```diff
@@ -39,12 +39,16 @@ export default class WidgetOptionsConfiguration extends Component {
         };
     }
 
+    componentWillReceiveProps(props) {
+        this.state.options = props.selectedWidget.props.configs.options;
+    }
+
     getCurrentWidgetOptionsInputs() {
         const CurrentWidgetOptionsInputs = [];
         const options = this.props.selectedWidgetConfiguration.configs.options;
         if (options) {
             for (let i = 0; i < options.length; i++) {
-                const value = this.state.options[options[i].id];
+                const value = this.state.options[options[i].id] || '';
```

Comment:
What if `this.state.options[options[i].id]` is false?

Reply:
When we passed ' ' to the material UI component, it will get  it as false and update correctly